core/src/services/gcs/backend.rs

```diff
@@ -66,6 +66,10 @@ pub struct GcsConfig {
     pub predefined_acl: Option<String>,
     /// The default storage class used by gcs.
     pub default_storage_class: Option<String>,
+    /// Explicitly disable authentication
+    ///
+    /// Used for testing purposes against storage emulators.
```

Comment:
Opendal doesn't expose features meant only for testing. We've added this feature because it proves useful. For exmaple, this is a public bucket that available for users to read and list. This does the use case we want to support.

So I believe this comment is wrong.

Reply:
Makes sense to me, I'll adjust this comment :+1: 


For reference, the [Iceberg Java impl](https://github.com/apache/iceberg/blob/6ee6d1327d3811dbd5795c4e87efdc41b7a58eaa/gcp/src/main/java/org/apache/iceberg/gcp/GCPProperties.java#L44-L45) mentions how this is useful for testing/storage emulator setups, so I followed similar here.

Reply:
Thank you for sharing your original idea with me.